paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PrimaryButtonUiStateMapperTest.kt

```diff
@@ -32,166 +32,157 @@ class PrimaryButtonUiStateMapperTest {
 
         val mapper = createMapper(
             isProcessingPayment = true,
-            currentScreenFlow = flowOf(PaymentSheetScreen.SelectSavedPaymentMethods),
-            buttonsEnabledFlow = flowOf(true),
-            amountFlow = flowOf(Amount(value = 1234, currencyCode = "usd")),
-            selectionFlow = flowOf(null),
-            customPrimaryButtonUiStateFlow = flowOf(usBankButton),
+            currentScreenFlow = stateFlowOf(PaymentSheetScreen.SelectSavedPaymentMethods),
+            buttonsEnabledFlow = stateFlowOf(true),
+            amountFlow = stateFlowOf(Amount(value = 1234, currencyCode = "usd")),
+            selectionFlow = stateFlowOf(null),
+            customPrimaryButtonUiStateFlow = stateFlowOf(usBankButton),
         )
 
         val result = mapper.forCompleteFlow()
 
         result.test {
             assertThat(awaitItem()).isEqualTo(usBankButton)
-            awaitComplete()
```

Comment:
state flows can never complete, so this assert wouldn't pass.

We don't actually rely on this behavior, or use it in our production codepaths, so it's safe to remove.

test also ensure we consumed all the events, so we're safe there.